external/include/mbedtls/aes.h

```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
  *
- * Copyright 2016 Samsung Electronics All Rights Reserved.
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
```

Comment:
This year means that Samsung starts this at 2016. It does not mean latest updated year.
So 2016 is correct. If this file is newly added, then 2018 is right.

Reply:
Thank you for informing me. Modifications completed.